xml/rmt_mirroring.xml

```diff
@@ -160,49 +161,66 @@ Jun 22 04:22:34 d31 systemd[1]: Started RMT Mirror timer.</screen>
     Example:
    </para>
 <screen>&prompt.sudo;<command>rmt-cli products list --all</command>
-+------+------------------+---------+--------------+----------------------+---------------+---------+---------------+
-| ID   | Name             | Version | Architecture | Product string       | Release stage | Mirror? | Last mirrored |
-+------+------------------+---------+--------------+----------------------+---------------+---------+---------------+
++------+----------------------+---------+--------+--------------+---------------
+| ID   | Product              | Version | Arch   | Mirror?      | Last mirrored
++------+----------------------+---------+--------+--------------+---------------
 [...]
-| 1743 | SUSE Package Hub | 15      | x86_64       | PackageHub/15/x86_64 | released      | false   |               |
+| 1743 | SUSE Package Hub     | 15      | x86_64 | Don't Mirror |
+|      | PackageHub/15/x86_64 |         |        |              |
 [...]
-+------+------------------+---------+--------------+----------------------+---------------+---------+---------------+
 
 &prompt.sudo;<command>rmt-cli product enable 1743</command>
 2 repo(s) successfully enabled.
 
 &prompt.sudo;<command>rmt-cli product disable 1743</command>
 2 repo(s) successfully disabled.</screen>
+<tip>
+ <title>Enabling and Disabling Multiple Products at Once</title>
+ <para>
+  To enable or disable multiple products at once, specify a space delimited
+  list of their IDs or product strings, for example:
+ </para>
+<screen>&prompt.sudo;<command>rmt-cli product enable 1743 SLES/15/x86_64 SLES/12</command></screen>
```

Comment:
Sorry, haven't seen this before, but here the same as with `repos` (vs. `repo`). The correct command namespace here is `products` (plural).

Please also check the rest of the docs for consistency :)